contracts/payloads/IGP40/PayloadIGP40.sol

```diff
@@ -0,0 +1,869 @@
+        require(
+            msg.sender == PROPOSER ||
+                msg.sender == TEAM_MULTISIG ||
+                address(this) == PROPOSER_AVO_MULTISIG ||
+                address(this) == PROPOSER_AVO_MULTISIG_2 ||
+                address(PROPOSER_AVO_MULTISIG_3) == PROPOSER_AVO_MULTISIG_3,
+            "msg.sender-not-allowed"
+        );
+
+        uint256 totalActions = 1;
+        address[] memory targets = new address[](totalActions);
+        uint256[] memory values = new uint256[](totalActions);
+        string[] memory signatures = new string[](totalActions);
+        bytes[] memory calldatas = new bytes[](totalActions);
+
+        // Action 1: call executePayload on timelock contract to execute payload related Fluid
+        targets[0] = address(TIMELOCK);
+        values[0] = 0;
+        signatures[0] = "executePayload(address,string,bytes)";
+        calldatas[0] = abi.encode(ADDRESS_THIS, "execute()", abi.encode());
+
+        uint256 proposedId = GOVERNOR.propose(
+            targets,
+            values,
+            signatures,
+            calldatas,
+            description
+        );
+
+        require(proposedId == PROPOSAL_ID, "PROPOSAL_IS_NOT_SAME");
+    }
+
+    function execute() external {
+        require(address(this) == address(TIMELOCK), "not-valid-caller");
+        
+        /// @notice Action 1: Set cbBTC token config and market rate curve on liquidity.
+        action1();
+
+        /// @notice Action 2: Config cbBTC/ETH and ETH/cbBTC
+        action2();
+
+        /// @notice Action 3: Config cbBTC/USDC and cbBTC/USDT
+        action3();
+
+        /// @notice Action 4: Config wstETH/cbBTC and weETH/cbBTC
+        action4();
+    }
+
+    function verifyProposal() external view {}
+
+    /***********************************|
+    |     Proposal Payload Actions      |
+    |__________________________________*/
+
+    /// @notice Action 1: Set cbBTC token config and market rate curve on liquidity.
+    function action1() internal {
+        {
+            AdminModuleStructs.RateDataV1Params[]
+                memory params_ = new AdminModuleStructs.RateDataV1Params[](1);
+
+            params_[0] = AdminModuleStructs.RateDataV1Params({
+                token: cbBTC_ADDRESS, // cbBTC
+                kink: 80 * 1e2, // 80%
+                rateAtUtilizationZero: 0, // 0%
+                rateAtUtilizationKink: 5 * 1e2, // 5%
+                rateAtUtilizationMax: 100 * 1e2 // 100%
+            });
+
+            LIQUIDITY.updateRateDataV2s(params_);
+        }
+
+        {
+            AdminModuleStructs.TokenConfig[]
+                memory params_ = new AdminModuleStructs.TokenConfig[](1);
+
+            params_[0] = AdminModuleStructs.TokenConfig({
+                token: cbBTC_ADDRESS, // cbBTC
+                threshold: 0.3 * 1e2, // 0.3
+                fee: 10 * 1e2, // 10%
+                maxUtilization: 100 * 1e2
+            });
+
+            LIQUIDITY.updateTokenConfigs(params_);
```

Comment:
we can skip this as it is the exact standard config that will be set via multisig through the new ListTokenAuth: https://github.com/Instadapp/fluid-contracts-public/blob/main/contracts/config/listTokenAuth/main.sol#L84